packages/jsii-pacmak/test/expected.jsii-calc-lib/java/src/main/java/software/amazon/jsii/tests/calculator/lib/IFriendly.java

```diff
@@ -12,9 +12,9 @@
     java.lang.String hello();
 
     /**
-     * A proxy class which for javascript object literal which adhere to this interface.
+     * A proxy class which represents a concrete javascript instance of this type.
      */
-    final class Jsii$Proxy extends software.amazon.jsii.JsiiObject implements software.amazon.jsii.tests.calculator.lib.IFriendly {
+    final static class Jsii$Proxy extends software.amazon.jsii.JsiiObject implements software.amazon.jsii.tests.calculator.lib.IFriendly {
```

Comment:
While you're at it, add the `sealed` (roughly equivalent to `final`) modifier to C# proxies. To do this, in `Amazon.JSII.Generator.Interface.InterfaceProxyGenerator`, change this line:

    SF.TokenList(SF.Token(SyntaxKind.InternalKeyword)),

to this

    SF.TokenList(SF.Token(SyntaxKind.InternalKeyword), SF.Token(SyntaxKind.SealedKeyword)),

And do the same for abstract class proxies.

Reply:
Added "sealed" for interface proxies.
I haven't implemented abstract class proxies for .NET (#223)

Reply:
So I am getting the following errors after adding `sealed`:

```
'VeryBasePropsProxy.Foo.set' is a new virtual member in sealed class 'VeryBasePropsProxy'
```

Reply:
reverted for now